autospec/build.py

```diff
@@ -153,17 +154,21 @@ def parse_buildroot_log(self, filename, returncode):
             return True
         self.must_restart = 0
         self.file_restart = 0
-        is_clean = True
+        fatal_buf = ''
         util.call("sync")
         with util.open_auto(filename, "r") as rootlog:
             loglines = rootlog.readlines()
         missing_pat = re.compile(r"^.*No matching package to install: '(.*)'$")
         for line in loglines:
             match = missing_pat.match(line)
             if match is not None:
-                util.print_fatal("Cannot resolve dependency name: {}".format(match.group(1)))
-                is_clean = False
-        return is_clean
+                if fatal_buf:
+                    fatal_buf = f"{fatal_buf}\nCannot resolve dependency name: {match.group(1)}"
```

Comment:
What about making fatal_buf a list instead so we can just append to it cleanly and join with LFs when we print?

Reply:
Oh nice, thanks!